api/services/wwwConfig.js

```diff
@@ -149,10 +149,15 @@ function _generateEnvs(bag, next) {
   var who = bag.who + '|' + _generateEnvs.name;
   logger.verbose(who, 'Inside');
 
-  var apiUrl = bag.apiSystemIntegration.data &&
-    bag.apiSystemIntegration.data.url;
+  var publicApiUrl = bag.publicApiIntegration.data &&
+    bag.publicApiIntegration.data.url;
+
+  var internalApiUrl = bag.internalApiIntegration.data &&
+    bag.internalApiIntegration.data.url;
+
+  internalApiUrl = internalApiUrl || publicApiUrl;
```

Comment:
This is probably not necessary any more.

Reply:
updated